docs/codacy-api/examples/running-dast-scans.md

```diff
@@ -0,0 +1,74 @@
+---
+description: Instructions on how to run DAST/App using the API.
+---
+
+# Run Dynamic Application Security Testing (DAST) scans
+
+Thanks to the new app scanning capabilities available on the Security and risk management dashboard, it's now possible to automate application scanning via Codacy's API. This means that, with little effort, you'll be able to run app scanning on demand every time you deploy a new version of your app.
+
+!!! important
+    App scanning is a business feature. If you are a Codacy Pro customer, contact our customer success team to access a short trial.
+    Check your permissions. Only git provider admins and organization managers will be able to create new targets and run scans (in app and via the API).
+    Get your API key. All the methods below require you to authenticate via your [account's API token](../api-tokens.md#account-api-tokens).
+
+## Creating targets
+
+Before the automation process itself, you need to create a target. Targets are single units that contain all the configurations of your scan, in this case the URL (and later on other configurations, like authentication, scan type or an OpenAPI definition).
+Targets only need to be defined once. The target's URL is immutable, so, if you need to change it, you'll need to delete the target and create a new one.
+
+To create a target:
+
+```bash
+curl -X POST https://app.codacy.com/api/v3/organizations/{GIT_PROVIDER}/{ORGANIZATION}/dast/targets \
+  -H "Content-Type: application/json" \
+  -H "Accept: application/json" \
+  -H "api-token: {API_KEY}" \
+  -d '{"url": "https://api.domain.com/v1"}'
+```
+
+Replace the placeholders with your own values:
+
+-   **API_KEY:** [Account API token](../api-tokens.md#account-api-tokens) used to authenticate on the Codacy API.
+-   **GIT_PROVIDER:** Git provider hosting of the organization, using one of the values in the table below. For example, `gh` for GitHub Cloud.
+
+    | Value | Git provider      |
+    |-------|-------------------|
+    | `gh`  | GitHub Cloud      |
+    | `ghe` | GitHub Enterprise |
+    | `gl`  | GitLab Cloud      |
+    | `gle` | GitLab Enterprise |
+    | `bb`  | Bitbucket Cloud   |
+    | `bbe` | Bitbucket Server  |
+
+-   **ORGANIZATION:** Name of the organization on the Git provider. For example, `codacy`. You must have admin permissions over the organization on the Git provider.
+
+Once you create the target you'll get the 'Target Id' as a response. That id will be used to trigger DAST scans in the next section.
+
+## Running DAST analysis scans
```

Comment:
It's funny that we mention 'trigger DAST scans in the next section' and then use the word running. I think we're using a bit of both and should stick to one nomenclature.